ddl/index.go

```diff
@@ -624,6 +641,7 @@ func (w *addIndexWorker) backfillIndexInTxn(handleRange reorgIndexTask) (nextHan
 		}
 		return nil
 	})
+	w.logSlowOperations(time.Since(oprStartTime), "backfillIndexInTxn", 3000)
```

Comment:
This is a constant threshold?

Reply:
I think 3s is large enough, so no needs to be configurable.